core/src/main/java/com/pholser/junit/quickcheck/runner/Shrinker.java

```diff
@@ -69,8 +70,7 @@ void shrink(
 
         ShrinkNode smallest =
             ShrinkNode.root(method, testClass, params, args, seeds, failure);
-        Queue<ShrinkNode> nodes = new ShrinkNodeQueue(smallest.magnitude());
-        nodes.addAll(smallest.shrinks());
+        Queue<ShrinkNode> nodes = new ArrayDeque<>(smallest.shrinks());
```

Comment:
Can we remove class `ShrinkNodeQueue`? If so, let's do so as part of this PR.

Reply:
Done